yb-voyager/cmd/exportSchema.go

```diff
@@ -766,9 +766,13 @@ func applyIndexFileTransformations() (*sqltransformer.IndexFileTransformer, erro
 	//assuming that assessment is run and fetched the redundant indexes
 	//TODO: see if we need to take care of the scenario where assessment is unable to fetch these
 	var err error
-	var redundantIndexToResolvedExistingIndex map[string]string
+	var redundantIndexToResolvedExistingIndex *utils.StructMap[*sqlname.ObjectNameQualifiedWithTableName, string]
 	redundantIndexToResolvedExistingIndex, err = fetchRedundantIndexMapFromAssessmentDB()
 	if err != nil {
+		if skipPerfOptimizations {
```

Comment:
can we have the check for skipPerfOptimizations at one place.
Preferably in cmd package, rather than having it in sqlfiletransformer package also.